docker/orthanc/AmazonLinux2Dockerfile

```diff
@@ -1,92 +1,55 @@
-
-########################## orthanc-runner-base
+##################
+# The dockerfile below is copied+adapted from docker/orthanc/Dockerfile.runner-base
+##################
 
 # base image to build and execute orthanc.
 # This image should contain only the packages that are common to both !
 # The purpose of this intermediate image is to benefit from Docker cache.
-ARG BASE_DEBIAN_IMAGE=bookworm-20250113-slim
-FROM debian:$BASE_DEBIAN_IMAGE AS orthanc-runner-base
-
-# we had some issues with one of the mirror -> force it to the Belgian mirror
-# RUN rm /etc/apt/sources.list.d/debian.sources
-# RUN echo "deb http://ftp.be.debian.org/debian/ bookworm main" | tee /etc/apt/sources.list.d/belgium.mirror.list
-
-RUN export DEBIAN_FRONTEND=noninteractive && \
-  apt-get --assume-yes update && \
-  apt-get --assume-yes install python3 && \
-  apt-get --assume-yes install python3-pip && \
-  apt-get --assume-yes install ca-certificates && \
-  apt-get --assume-yes install tzdata && \
-  apt-get --assume-yes install locales && \
-  apt-get --assume-yes install libcurl4 && \
-  apt-get --assume-yes install libdcmtk17 && \
-  apt-get --assume-yes install libboost-log1.74.0 libboost-filesystem1.74.0 libboost-locale1.74.0 libboost-regex1.74.0 libboost-system1.74.0 libboost-thread1.74.0 libboost-iostreams1.74.0 && \
-  apt-get --assume-yes install libjpeg62-turbo && \
-  apt-get --assume-yes install libjsoncpp25 && \
-  apt-get --assume-yes install liblua5.4 && \
-  apt-get --assume-yes install libpng16-16 && \
-  apt-get --assume-yes install libpugixml1v5 && \
-  apt-get --assume-yes install libsqlite3-0 && \
-  apt-get --assume-yes install libssl3 && \
-  apt-get --assume-yes install libuuid1 && \
-  apt-get --assume-yes install lsb-base && \
-  apt-get --assume-yes install zlib1g && \
-  apt-get --assume-yes install libpq5 && \
-  apt-get --assume-yes install libmariadb3 && \
-  apt-get --assume-yes install unixodbc && \
-  apt-get --assume-yes install libopenslide0 && \
-  apt-get --assume-yes install libcrypto++ && \
-  apt-get --assume-yes install libcpprest && \
-  apt-get --assume-yes install libprotobuf32 && \
-  apt-get --assume-yes clean && \
-  rm --recursive --force /var/lib/apt/lists/*
+FROM 707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/amazonlinux-base:master AS orthanc-runner-base
+
+RUN dnf -y update && \
+    dnf -y install python3.11-devel \
+    ca-certificates tzdata \
+    libcurl-devel boost-devel libjpeg-turbo-devel \
+    jsoncpp-devel lua-devel libpng-devel sqlite-devel \
+    openssl-devel libuuid-devel unixODBC-devel \
+    protobuf-devel && \
+    dnf clean all
 
 # Make sure the en_US locale has been generated (required for
 # case-insensitive comparison of strings with accents)
-RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
-RUN locale-gen
+# (pauline 02/2025) The lines below don't work, but everything seems to work fine without this
```

Comment:
Can you explain what you mean by this? Why don't they work?

Reply:
I tried to translate this:
```
RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
RUN locale-gen
```
to an amazon linux install, but the installation doesn't work. I commented it out to possibly get back to it later, but everything works fine without it